test/e2e/framework/helpers.go

```diff
@@ -168,6 +170,24 @@ func (f *Framework) MustGetPods(t *testing.T, namespace string) *v1.PodList {
 	return pods
 }
 
+func (f *Framework) IsFeatureGateEnabled(t *testing.T, name string) bool {
+	t.Helper()
+	fg, err := f.OpenShiftConfigClient.ConfigV1().FeatureGates().Get(context.Background(), "cluster", metav1.GetOptions{})
+	if err != nil {
+		t.Fatalf("failed to get featuregate `cluster`: %s", err.Error())
+	}
+
+	enabledFeatureGates := fg.Status.FeatureGates[0].Enabled
```

Comment:
just curious, is there always one and only one `featuregate` resource in every cluster?

Reply:
For `cluster` I have seen only. Not sure why it is an array though :)

Reply:
https://github.com/openshift/api/blob/081317a013f87bfe55655c0441c07cfbd7087c54/config/v1/types_feature.go#L85-L102

There can be multiple `FeatureGates` and iiuc they should differ by version. 
Reading the doc string I do wonder if we need to actually look at the versions here (and in the operator code too)?

Reply:
I suppose this could list feature gates for multiple versions that change the clusters behavior when the cluster is updated...

Reply:
updated code to include the version check as well